app/templates/app/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'app/base.html' %}
+{% block content %}
+    <h2>Login User</h2>
+    <form method="post" novalidate>
+        {% csrf_token %}
+        <div class="form-group">
```

Comment:
make sure to show form field errors as well.

Reply:
Done sir